### PR TITLE
[BUGFIX][CU-86drved4n] Surveys: redirect to the correspondent thank you page based on origin

### PR DESCRIPTION
**Pull request by cgarcia-lightit**
# Light It Pull Request Template ⚡⚡⚡

_Resolve_: [CU-86drved4n](https://app.clickup.com/t/86drved4n)

## Description and Requires ⭐
bugfix: render the rights faqs using the flows data

## Checklist ✅

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by martinopp** [comment]
Task linked: [CU-86drved4n Surveys: redirect to the correspondent thank you page based on origin](https://app.clickup.com/t/86drved4n)

File: apps/eo_web/src/components/FAQs.tsx [hunk 1]
```diff
@@ -23,7 +24,7 @@ export const FAQs = ({ usePayment = false, channel, flow }: FAQsProps) => {
   ];
   let faqList = faqs;
   if (flow && flowsWithCOrgFaqs.includes(flow)) faqList = cOrgFaqs;
-  else if (channel === "cancer" && !usePayment) faqList = pilotFaqs;
+  else if (channel === "cancer" && pilot) faqList = pilotFaqs;
```
Comment (sgoycoechea-lightit) [inline apps/eo_web/src/components/FAQs.tsx:27]: Now that we have the `flow`, we can remove the `pilot` boolean and just test that `flow === cancer_pilot`

File: apps/eo_web/src/screens/ProfilingThankYou.tsx [hunk 2]
```diff
@@ -102,7 +102,7 @@ export const ProfilingThankYou = () => {
       <FAQs
         channel={channel ?? undefined}
         flow={flow}
-        usePayment={usePayment}
+        pilot={flow == "cancer_pilot"}
```
Comment (sgoycoechea-lightit) [inline apps/eo_web/src/screens/ProfilingThankYou.tsx:105]: We don't need to pass the `pilot` anymore, we can just pass the `flow`.

File: apps/eo_web/src/screens/Cancer/CancerSurveyThankYou.tsx [hunk 3]
```diff
@@ -72,8 +71,12 @@ export const CancerSurveyThankYou = () => {
...
         </Typography>
       </AllDonePanel>
-      <HowEOWorks pilot={!usePayment} />
-      <FAQs channel="cancer" flow={flow} usePayment={usePayment} />
+      <HowEOWorks pilot={pilot && flow === Flows.cancer_pilot} />
+      <FAQs
+        channel="cancer"
+        flow={flow}
+        pilot={pilot && flow !== Flows.marketing_site}
+      />
```
Comment (sgoycoechea-lightit) [inline apps/eo_web/src/screens/Cancer/CancerSurveyThankYou.tsx:79]: Remove `pilot` prop

**Pull request merged**